frontend/www/js/omegaup/course/edit.ts

```diff
@@ -172,16 +173,47 @@ OmegaUp.on('ready', () => {
               })
               .catch(ui.apiError);
           },
+          'get-versions': (
+            problemAlias: string,
+            source: course_ProblemList,
+          ) => {
+            api.Problem.versions({ problem_alias: problemAlias })
+              .then(function (result) {
+                source.versionLog = result.log;
+                let currentProblem = null;
+                for (const problem of source.problems) {
+                  if (problem.alias === problemAlias) {
+                    currentProblem = problem;
+                    break;
+                  }
+                }
+                let publishedCommitHash = result.published;
+                if (currentProblem != null) {
+                  publishedCommitHash = currentProblem.commit;
+                }
```

Comment:
```suggestion
                let publishedCommitHash = result.published;
                for (const problem of source.problems) {
                  if (problem.alias === problemAlias) {
                    publishedCommitHash = problem.commit;
                    break;
                  }
                }
```

como `currentProblem` sólo se usa para obtener `publishedCommitHash`, mejor podemos darle inline para simplificar el código.

Reply:
Me agrada el cambio. Ya quedó listo.